GroovyRP/Program.cs

```diff
@@ -1,191 +1,93 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Diagnostics;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
 using DiscordRPC;
-using CSCore;
-using CSCore.CoreAudioAPI;
-using TagLib;
-using CsvHelper;
-using System.IO;
-using System.Globalization;
-using System.ComponentModel;
+using DiscordRPC.Message;
 
 namespace GroovyRP
 {
     class Program
     {
-        public static DiscordRpcClient client;
+        private const string appDetails = "GroovyRP\nOrignal Creator: https://github.com/dsdude123/GroovyRP \nThis Version: https://github.com/DraxCodes/GroovyRP\n\n";
```

Comment:
Please remove reference to forked repository.

Reply:
whoops, didn't realise i left this in. Done